Doc/library/subprocess.rst

```diff
@@ -689,7 +689,10 @@ execute, will be re-raised in the parent.
 
 The most common exception raised is :exc:`OSError`.  This occurs, for example,
 when trying to execute a non-existent file.  Applications should prepare for
-:exc:`OSError` exceptions.
+:exc:`OSError` exceptions. Note that, when ``"shell=True"``, :exc:`OSError`
```

Comment:
IMO the code fragment should not be a string

Reply:
Darn, you are right. See #27297 / bpo-44713